src/ServiceControl/Recoverability/Grouping/Retries/RetryOperationCompletedHandler.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace ServiceControl.Recoverability
+{
+    using NServiceBus;
+    using Raven.Client;
+    using ServiceBus.Management.Infrastructure.Settings;
+    using System.Linq;
+
+    public class RetryOperationCompletedHandler : IHandleMessages<RetryOperationCompleted>
+    {
+        private readonly Settings settings;
+        public IDocumentStore Store { get; set; }
+        public RetryOperationManager RetryOperationManager { get; set; }
+
+        public void Handle(RetryOperationCompleted message)
+        {
+            var completedOperation = new CompletedRetryOperation
+            {
+                RequestId = message.RequestId,
+                RetryType = message.RetryType,
+                CompletionDate = message.CompletionDate
+            };
```

Comment:
We don't store or act on this being a failed operation or not. Is this what we want if we are showing this in the UI as "LAST 10 SUCCESSFUL GROUP RETRIES"? I would expect either to only see successfully completed retries here or a different header like "LAST 10 GROUP RETRIES".